### PR TITLE
add section: A 'Shit' Example

### PR DESCRIPTION
**Pull request by chantastic**
This section serves to provide a semi-humorous and complete simple example.

I think that this is enough to help most understand the **how** of Context in a simple case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,53 @@
...
+let ContextualExclamation = () => (
+  <ExpletiveContext.Consumer>
+    {word => <span>Oh {word}!</span>}
+  </ExpletiveContext.Consumer>
+);
+
+// Grandma *hates* the word "shit".
+// Say "poop" at Grandma's house.
+let GrandmasHouse = props => (
+  <ExpletiveContext.Provider value="poop" {...props} />
```
Comment (gaearon) [inline README.md]: Most people don't realize `{...props}` includes children until they've specifically thought about this so they might not get how nesting works.

Comment (gaearon) [reply README.md]: It's also not very clear from this example how this is different from just passing props. Maybe a component in between would help.

Comment (chantastic) [reply README.md]: I agree. Do you have examples in the docs or elsewhere that do a good job of illustrating deeply nested components without being contrived? I have a contrived example in the "create, consume, provide" section.

Would it be fair to link to a CodeSanbox with a more "real-world" example?

Comment (gaearon) [reply README.md]: This isn't a bad one I think. You can see there's a thing in between

screen shot 2018-04-03 at 12 40 15 am

Comment (chantastic) [reply README.md]: Cool. I plan to continue with the "bare-bones" example, with a follow-up example that introduces the intermediate components.

**Pull request merged**